libpod/image/prune.go

```diff
@@ -44,6 +46,10 @@ func (ir *Runtime) PruneImages(ctx context.Context, all bool) ([]string, error)
 	}
 	for _, p := range pruneImages {
 		if err := p.Remove(ctx, true); err != nil {
+			if errors.Cause(err) == storage.ErrImageUsedByContainer {
+				logrus.Warnf("Failed to prune image %s as it is in use: %v", p.ID(), err)
```

Comment:
In podman rmi we do this:
```
fmt.Printf("A container associated with containers/storage, i.e. via Buildah, CRI-O, etc., may be associated with this image: %-12.12s\n", img.ID())
```
could we massage this message to be similar to give the user a heads up on what the issue is?   FWIW, in my latest PR I'm changing that message to:

```
fmt.Printf("A container associated with containers/storage, i.e. via Buildah, CRI-O, etc., may be associated with this image: %-12.12s\nUsing the --force option will remove the container and image, but may cause failures for other dependent systems.", img.ID())
```

Which is longer than I'd like, but at least tells the user how to get around the issue.

Reply:
@TomSweeneyRedHat Could you update your latest PR to fix this the way you want as well.
I will merge this and then we can fix the messages in your PR.

Reply:
@rhatdan ack